doc/users/next_whats_new/widget_button_styling.rst

```diff
@@ -0,0 +1,32 @@
+Custom styling of Button widgets
+--------------------------------
+
+Additional custom styling of button widget may be achieved via the
+*label_props* and *radio_props* arguments to `.RadioButtons`; and the
+*label_props*, *frame_props*, and *check_props* arguments to `.CheckButtons`.
+
+.. plot::
+
+   from matplotlib.widgets import CheckButtons, RadioButtons
```

Comment:
These don't look great:
![image](https://user-images.githubusercontent.com/6197628/213554818-de600705-ee27-40c7-8075-889c4b835358.png)
Is it possible to do a tight layout on them, or manually adjust the subplot parameters so the text is in bounds of the figure?

Reply:
Our buttons unfortunately do not do any constrained layout; I just picked a random font size to show it works, but this obviously doesn't look great.

Reply:
Ah, I didn't realize the Axes title was cropped as well; I thought you had just screenshotted that way. I'll fix that.